routes/s3upload.js

```diff
@@ -0,0 +1,40 @@
+const s3router = require('express-promise-router')();
+const aws = require('aws-sdk');
```

Comment:
🚫 **[eslint]** <[import/no-unresolved](https://github.com/import-js/eslint-plugin-import/blob/v2.25.2/docs/rules/no-unresolved.md)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Unable to resolve path to module 'aws-sdk'.